### DISCUSSION TITLE
EE1D02FF-AA32-416E-98CA-016E38A81E8E

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#75] [General]
# EE1D02FF-AA32-416E-98CA-016E38A81E8E

Gliese 163 is a nearby red dwarf star about 49.36 light-years away in the constellation Dorado. It has a total of five planets, all neptunian, with distances ranging from 0.06 to 1.02 AU from their host star

https://celestia.mobi/addon?item=EE1D02FF-AA32-416E-98CA-016E38A81E8E&lang=en

**Comment by exocritic5457** [comment]
this should definitely be remade, but the unfortunate part is I have little to no idea on how to go about such a thing at the moment.

**Reply by Karistus** [reply]
Its on the list, dont worry

**Reply by Galaxy-Leaper1** [reply]
Pull request has been created, please stay tuned

**Reply by exocritic5457** [reply]
I mean... they aren't neon green, I'll give you that

**Reply by exocritic5457** [reply]
I retract my previous statement. This 'update' deeply saddens my being.

**Reply by Galaxy-Leaper1** [reply]
Sorry, it didn't work out as you wanted, but if you can, you can even make a brand new add-on for it
